src/test/groovy/org/prebid/server/functional/util/PBSUtils.groovy

```diff
@@ -97,4 +96,9 @@ class PBSUtils implements ObjectMapperWrapper {
     static BigDecimal getRandomPrice(int min = 0, int max = 10, int scale = 3) {
         getRandomDecimal(min, max).setScale(scale, HALF_UP)
     }
+
+    static int getRandomNumberWithExcept(int exceptValue, int min = 0, int max = MAX_VALUE) {
```

Comment:
- please move this method closer to the `getRandomNumber`
- `getRandomNumberWithExcept` -> `getRandomNumberWithExclusion`